src/sys/fuchsia/eventedfd.rs

```diff
@@ -0,0 +1,187 @@
+use {io, poll, Evented, Ready, Poll, PollOpt, Token};
+use libc;
+use magenta;
+use magenta::HandleBase;
+use sys::fuchsia::{DontDrop, poll_opts_to_wait_async, sys};
+use std::mem;
+use std::os::unix::io::RawFd;
+use std::sync::{Arc, Mutex};
+
+/// Properties of an `EventedFd`'s current registration
+#[derive(Debug)]
+pub(in sys::fuchsia) struct EventedFdRegistration {
```

Comment:
Yeah, that's true-- it's not exported from `fuchsia/mod.rs`, so I guess I can change it. During development, I wanted to make sure I wasn't exporting it publicly since you can trigger unsafety by swapping out the `RawFd` (which is closed on `Drop`).